src/components/tables/TableActions.ts

```diff
@@ -0,0 +1,62 @@
+import { IData } from './Table';
+import { ITableState, ITableData } from './TableReducers';
+import { IReduxAction } from '../../utils/ReduxUtils';
+
+export enum MODIFY_SATE_TABLE { MODIFY_SATE_TABLE = 'MODIFY_STATE_TABLE' };
+
+export type ITableStateModifyer = (state: ITableState) => ITableState;
+
+export const TableActions = {
+  add: 'ADD_TABLE',
+  remove: 'REMOVE_TABLE',
+  inError: 'IN_ERROR_TABLE',
+  toggleLock: 'TOGGLE_LOCK_TABLE',
+  modifyState: MODIFY_SATE_TABLE.MODIFY_SATE_TABLE,
```

Comment:
Why the enum for this one? If it is necessary, I would change its name.

Reply:
it's not, will remove